lib/greed/score.rb

```diff
@@ -0,0 +1,24 @@
+module Greed
+  def calcScore(dice)
+    raise ArgumentError.new unless dice
+    score = 0
+
+    # Create hash of count of each die
+    counts = dice.inject(Hash.new(0)) {|hash, num| hash[num] += 1; hash}
```

Comment:
I would not let this line of code in to a project that I was working on. It's not immediately clear what's happening. See if you can find a way to make it more verbose (through code) rather than relying on a single complicated line.